gemfiles/prod/Gemfile.lock

```diff
@@ -458,7 +458,7 @@ GEM
     net-http-digest_auth (1.4)
     net-http-persistent (2.9.4)
     netrc (0.11.0)
-    newrelic_rpm (5.6.0.349)
+    newrelic_rpm (5.7.0.350)
```

Comment:
We need to merge https://github.com/3scale/porta/pull/620 separately first, which does exactly this.